lib/Log/Any/Proxy.pm

```diff
@@ -55,6 +55,10 @@ sub new {
         require Carp;
         Carp::croak("$class requires a 'context' parameter");
     }
+    unless ( $self->{hooks} ) {
```

Comment:
I followed the style of the other parameters `adapter`, `category` and `context`.
Besides that, this eliminates the need to check if the parameter exists every time.
Alternatively, of course, we could create `hooks` in `init()`.
But if hooks is a feature of `Log::Any`, then it would be supported in every Proxy, right?